src/common/options/mgr.yaml.in

```diff
@@ -292,6 +292,14 @@ options:
   default: true
   services:
   - mgr
+- name: mon_warn_on_pool_no_app_interval
```

Comment:
Just a suggestion, but `mon_warn_on_pool_no_app_grace` might be a better name -- by analogy with e.g. `mon_down_mkfs_grace` option.